src/components/SignInButtons/AppleSignIn/index.desktop.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {View} from 'react-native';
+import withLocalize, {withLocalizePropTypes} from '../../withLocalize';
+import ButtonBase from '../ButtonBase';
+import AppleLogoIcon from '../../../../assets/images/signIn/apple-logo.svg';
+import CONFIG from '../../../CONFIG';
+import ROUTES from '../../../ROUTES';
+
+const propTypes = {...withLocalizePropTypes};
+
+const $appleButtonContainerStyle = {
```

Comment:
Note: Expensify will likely want _all_ styles moved into the `src/styles.js` file.